File: persistencia/src/main/java/co/edu/uniquindio/proyecto/dto/ComentarioGetDTO.java
package co.edu.uniquindio.proyecto.dto;

import co.edu.uniquindio.proyecto.entidades.Comentario;
import co.edu.uniquindio.proyecto.entidades.Producto;
import co.edu.uniquindio.proyecto.entidades.Usuario;
import lombok.*;

import java.time.LocalDateTime;

@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class ComentarioGetDTO {
    private String texto;
    private LocalDateTime fecha;
    private String usuarioCedula;
    private Integer productoCodigo;

    private Double calificacion;

}

File: my-app/src/app/modelo/comentario-get-dto.ts
export class ComentarioGetDTO {
  texto: String = '';
  fecha: Date = new Date();
  usuarioCedula: string = '';
  productoCodigo: number = 0;
  calificacion: number = 0;
  constructor(
    texto: string,
    fecha: Date,
    usuarioCedula: string,
    productoCodigo: number,
    calificacion: number
  ) {
    this.texto = texto;
    this.fecha = fecha;
    this.usuarioCedula = usuarioCedula;
    this.productoCodigo = productoCodigo;
    this.calificacion = calificacion;
  }
}
